Add docs for github app agent for execution (#163)
port-labs/port-docs: docs/exporters/github-exporter/self-hosted-installation.md

:::note Prerequisites

> - You will need your Port `CLIENT_ID` and `CLIENT_SECRET`. To find your Port API credentials go to [Port](https://app.getport.io), click on `Credentials` in the bottom left corner, and you will be able to view and copy your `CLIENT_ID` and `CLIENT_SECRET`:

Reviewer: ```suggestion
- You will need your Port `CLIENT_ID` and `CLIENT_SECRET`. To find your Port API credentials go to [Port](https://app.getport.io), click on `Credentials` at the bottom left corner, and you will be able to view and copy your `CLIENT_ID` and `CLIENT_SECRET`:
```

| `APP_ID`             | Application ID, it should show up in the edit GitHub App page in the upper section                                                  |
| `WEBHOOK_SECRET`     | The same string that was been used to register the application in the previous step                                                 |
| `GHE_HOST`           | Your organization's self-hosted GitHub hostname                                                                                     |
| `PORT`               | The port that the GitHub App will listen to                                                                                         |
| `PORT_URL`           | Port's API Base URL                                                                                                                 |
| `PORT_CLIENT_ID`     | Port client id for interacting with the API                                                                                         |
| `PORT_CLIENT_SECRET` | Port client secret for interacting with the API                                                                                     |

> | `PRIVATE_KEY`        | A base64 encoded private key, you can get the private key base64 value by running the command `base64 -i private-key-file-path.pem` |

Reviewer: ```suggestion
| `PRIVATE_KEY`        | A base64 encoded private key. Recieve it by running the command `base64 -i private-key-file-path.pem` |
```
